<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="L4024.local" name="runners.TestRunner" tests="14" failures="0" timestamp="01 Aug 2019 17:58:10 GMT" time="348.612" errors="1">
  <testcase name="runScenario" time="24.420" classname="runners.TestRunner"/>
  <testcase name="runScenario" time="33.479" classname="runners.TestRunner"/>
  <testcase name="runScenario" time="44.738" classname="runners.TestRunner"/>
  <testcase name="runScenario" time="18.915" classname="runners.TestRunner"/>
  <testcase name="runScenario" time="17.470" classname="runners.TestRunner"/>
  <testcase name="runScenario" time="31.960" classname="runners.TestRunner"/>
  <testcase name="runScenario" time="28.089" classname="runners.TestRunner"/>
  <testcase name="runScenario" time="18.998" classname="runners.TestRunner"/>
  <testcase name="runScenario" time="8.389" classname="runners.TestRunner"/>
  <testcase name="runScenario" time="19.347" classname="runners.TestRunner"/>
  <testcase name="runScenario" time="27.103" classname="runners.TestRunner"/>
  <testcase name="runScenario" time="24.676" classname="runners.TestRunner">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at pageObjects.AccountHomePage.isAccountHomePageOpened(AccountHomePage.java:25)
	at stepDefinitions.LoginSteps.login_should_be_successful(LoginSteps.java:39)
	at âœ½.login should be successful(features/login/Login.feature:13)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="44.335" classname="runners.TestRunner"/>
  <testcase name="runScenario" time="6.693" classname="runners.TestRunner"/>
</testsuite> <!-- runners.TestRunner -->
